docs/development/getting-started.rst

```diff
@@ -17,13 +17,32 @@ installed with ``pip``.
     $ # Specify your Python version here.
     $ nox -e tests -p py310
 
+
+Rust dependencies on Linux
```

Comment:
This is duplicative of the extensive docs we have in the installation page (which covers many platforms).

It'd be better to just link https://cryptography.io/en/latest/installation/#rust

Reply:
Yes. True. Thanks.

I now see https://cryptography.io/en/latest/installation/#rust
I missed it on the first read... as I was focued on the dev part ... install would come later :)

Reply:
The thing is that I tried `cargo` via apt-get install on Ubuntu, but it was intalled without `fmt` or `clippy` or `llvm-tools` 
so `nox -e rust` was failing.